extensions/scarb-cairo-test/src/main.rs

```diff
@@ -32,6 +32,10 @@ struct Args {
     /// Whether to print resource usage after each test.
     #[arg(long, default_value_t = false)]
     print_resource_usage: bool,
+
+    /// Do not rebuild the package.
+    #[arg(long, default_value_t = false)]
+    no_build: bool,
```

Comment:
What is the purpose of this feature? It brings a risk of testing out of date code